src/main/java/org/radarbase/output/kafka/S3KafkaStorage.kt

```diff
@@ -0,0 +1,48 @@
+package org.radarbase.output.kafka
+
+import io.minio.MinioClient
+import org.apache.avro.file.SeekableFileInput
+import org.apache.avro.file.SeekableInput
+import org.radarbase.output.util.TemporaryDirectory
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.Paths
+
+
+class S3KafkaStorage(
+        private val s3Client: MinioClient,
```

Comment:
I assume the minio client is compatible with all S3 compatible services?

Reply:
Indeed, at least it's tested with MinIO and AWS S3.